packages/block-library/src/list/index.js

```diff
@@ -220,51 +225,7 @@ export const settings = {
 		};
 	},
 
-	edit( {
-		attributes,
-		insertBlocksAfter,
-		setAttributes,
-		mergeBlocks,
-		onReplace,
-		className,
-	} ) {
-		const { ordered, values } = attributes;
-
-		return (
-			<RichText
-				identifier="values"
```

Comment:
Sometimes we seem to set these as constants, other times not. Would it be good to standardise this too? Personally, I feel like the constants are good, but a separate file for it a bit too much.

Reply:
Yes, we should both ensure they are provided and figure out what's the best way to encode them. We can't use constants inside `block.json` files but maybe it would be possible to create ESLint rule which would ensure that identifiers use the existing attribute's name.